testapi.pm

```diff
@@ -2283,8 +2283,14 @@ A typical call would look like:
 sub compat_args {    # no:style:signatures
     my ($def_args, $fix_keys) = splice(@_, 0, 2);
     my %ret;
-    for my $key (@{$fix_keys}) {
-        $ret{$key} = shift if (scalar(@_) >= 1 && (!defined($_[0]) || !grep { $_ eq $_[0] } keys(%{$def_args})));
+    if (scalar(@$fix_keys) == 1) {
+        if ((scalar(@_) % 2) != 0) {
+            $ret{$fix_keys->[0]} = shift(@_);
```

Comment:
```suggestion
            $ret{$fix_keys->[0]} = shift;
```

Reply:
and I changed it to use `signatures` where we do not have this implicit shift...